sdk/pinocchio/src/sysvars/epoch_rewards.rs

```diff
@@ -0,0 +1,50 @@
+use super::Sysvar;
+use crate::impl_sysvar_get;
+
+pub const HASH_BYTES: usize = 32;
+
+#[derive(Default, Debug, Clone)]
+pub struct Hash([u8; HASH_BYTES]);
+
+impl Hash {
+    pub fn to_bytes(self) -> [u8; HASH_BYTES] {
+        self.0
+    }
+}
```

Comment:
I'm not an expert in how things are organized in Pinocchio, but it seems strange to define the blockhash type inside `sysvars::epoch_rewards`.

Reply:
Having a dedicated type for the hashes seems quite reasonable.
Not sure what is the state of the Pinocchio coverage here.
There should be a `Hash` type at some point, I would think.